macros/parserGraphTool.pl

```diff
@@ -228,6 +235,17 @@ sub new {
 }
 
 our %graphObjectTikz = (
+	point => {
+		code => sub {
+			my $self = shift;
+			my ($x, $y) = @{$_->{data}[2]{data}};
+			my $point = "($x,$y)";
+			return ("\\draw[thick,blue,fill=blue!30] $point circle[radius=5pt];", [
```

Comment:
This is just a suggestion, but to make this point look more like the point in the javascript graph, you could change this to
```suggestion
			return ("\\draw[line width=4pt,blue,fill=red] $point circle[radius=5pt];", [
```